docs/guide-send-tx-bundle.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Guide to Sending Transactions and Bundles on Flashbots
+---
+
+Navigating the various methods for submitting transactions and bundles on Flashbots can be overwhelming. This guide aims to clarify the available options, helping you choose the most suitable one.
+
+## Understanding Relay and RPC Endpoints
+
+Flashbots offers two primary JSON-RPC endpoints on the Ethereum Mainnet: `rpc.flashbots.net` and `relay.flashbots.net`.
+
+### `rpc.flashbots.net` - For Retail Users
+- **Purpose**: Designed specifically for retail users as drop-in RPC replacement for MEV protection and refund.
+- **Features**: Provides MEV protection and MEV refunds for eligible transactions. Support all the regular [Ethereum JSON RPC methods](https://docs.infura.io/networks/ethereum/json-rpc-methods/) but not [Flashbots specific JSON RPC
+methods](flashbots-auction/advanced/rpc-endpoint).
+- **Performance**: [Rate limited](/flashbots-protect/additional-documentation/ratelimiting) to satisfy the need of regular users.
+- **Note**: Transactions are submitted through `eth_sendRawTransaction` to fit wallet RPC interface. Underneath, the endpoint uses `relay.flashbots.net` to submit transactions.
+
+### `relay.flashbots.net` - For Advanced Users
+- **Purpose**: Designed for advanced users, including searchers and Telegram bots, to submit transactions and bundles.
```

Comment:
I'd mention applications as a user. Eg "Designed for advanced users, including searchers, applications, and Telegram bots."

At that point the sentence is fairly long so you could end it there and add a new one for "accepts transactions and bundles" or something like that.